README.md

```diff
@@ -23,20 +21,42 @@ The Starbase platform's smart contracts allow investors who hold STARs to partic
 This contract creates a token with customized name, symbol and decimals. Upon its creation, the token transfer functionality is paused, that is to say no one is able to trade them. This is possible to be reverted only by the token contract owner - who is the Ethereum address that deployed the token mold contract.
 For this contract to work with `TokenSale.sol`, its ownership needs to be passed on to `TokenSale` contract instance that will manage the token sale.
 
+Note: TokenSale does not need to have this smart contract to operate. It accepts any ERC20 smart contract
+
 -   Whitelist.sol
 
 It allows the addition and/or removal of addresses to the whitelist registry. Both addition and removal of addresses is done via passing an array of Ethereum addresses to `addManyToWhitelist` and `removeManyFromWhitelist` functions within the contract. Only the contract owner has the ability to trigger such actions.
 
+-   FundsSplitter.sol
+
+Contract that splits TokenSale funds between Starbase and client. Once deployed, it can serve as `wallet` parameter from the TokenSale contract by adding its the FundsSplitter address in as the `wallet` parameter in the TokenSale `init` function.
```

Comment:
```suggestion
Contract that splits TokenSale funds between Starbase and client. Once deployed, it can serve as `wallet` parameter from the TokenSale contract by adding its address in as the `wallet` parameter in the TokenSale `init` function.
```